tests/pyreverse/data/classes_colorized.puml

```diff
@@ -0,0 +1,42 @@
+@startuml classes_colorized
+set namespaceSeparator none
+class "Ancestor" as data.clientmodule_test.Ancestor #aliceblue {
+attr : str
+cls_member
+
+get_value()
+set_value(value)
+}
+class "<color:red>CustomException</color>" as data.suppliermodule_test.CustomException #aliceblue {
+
+}
+class "DoNothing" as data.suppliermodule_test.DoNothing #aliceblue {
+
+}
+class "DoNothing2" as data.suppliermodule_test.DoNothing2 #aliceblue {
+
+}
+class "DoSomething" as data.suppliermodule_test.DoSomething #aliceblue {
+my_int : Optional[int]
+my_string : str
+
+do_it(new_int: int) -> int
+}
+class "Interface" as data.suppliermodule_test.Interface #aliceblue {
+
+
+get_value()
+set_value(value)
+}
+class "Specialization" as data.clientmodule_test.Specialization #aliceblue {
+TYPE : str
+relation
+relation2
+top : str
+}
```

Comment:
```suggestion
class "Specialization" as data.clientmodule_test.Specialization #aliceblue {
    TYPE : str
    relation
    relation2
    top : str
}
```
I probably missed it in the last review but I think it would be nice to have proper indentation in generated diagrams. I think it might be modified a little after generation so it would help.

Reply:
Yes, that would definitely make it more readable! ``VCGPrinter`` already had methods for handling the indentation, I moved them into the base ``Printer`` class and reused them in ``PlantUmlPrinter``.